### ISSUE TITLE
Packageerror with update to Debian-Buster

### ISSUE THREAD

**Issue by centertivevier** [Icinga/icinga-packaging#135]
Hi there,

i would be update my system to "Debian-Buster" but if they change the sourcelist to:

```
deb http://packages.icinga.com/debian icinga-buster main
deb-src http://packages.icinga.com/debian icinga-buster main
```
there get the following error:

`E: Fehlschlag beim Holen von http://packages.icinga.com/debian/dists/icinga-buster/InRelease  Erwarteter Eintrag »main/source/Sources« konnte in Release-Datei nicht gefunden werden.`

**Comment by dnsmichi** [comment]
AFAIK Debian Buster is not released yet, points to 6.7.2019.

**Comment by lazyfrosch** [comment]
This only affected the `deb-src` part, which is optional btw.

I re-published the repositories including the source now.
